Buildscripts/CDN/generate-files.py

```
@@ -0,0 +1,198 @@
+    if not os.path.exists(in_path):
+        exit_with_error(f"Input path not found: {in_path}")
+    if os.path.exists(out_path):
+        shutil.rmtree(out_path)
+    os.mkdir(out_path)
+    mapping = read_mapping_file()
+    device_directories = os.listdir(in_path)
+    device_index = DeviceIndex(version, [])
+    for device_directory in device_directories:
+        if not device_directory.endswith("-symbols"):
+            device_id = device_directory[10:]
+            if not device_id in mapping.sections():
```

Ruff (0.14.2)

Test for membership should be `not in`

Convert to `not in`

(E713)